device/src/esp32_run.rs

```diff
@@ -82,7 +84,7 @@ where
         };
 
         // The event log gets the reset of the sectors
-        let mut event_log = EventLog::new(partitions.nvs);
+        let mut event_log = EventLog::new(share_partition, partitions.nvs);
```

Comment:
We should rename `EventLog`, maybe `DataPartitions` or `DeviceDatabase` `DeviceData`.

Reply:
I was looking at this but in the end the API is just an event log. I'll change  it to MutationLog.